openedx/adg/lms/applications/apps.py

```diff
@@ -0,0 +1,11 @@
+"""
+All configurations for applications app
+"""
+from django.apps import AppConfig
+
+
+class ApplicationsConfig(AppConfig):
```

Comment:
config is defined but not overriding the default config, so this configuration is not being used, we need to override in admin.